### DISCUSSION TITLE
I have copilot from github for education, but I do not get it in my IDE or on the copilot settings tab

### DISCUSSION THREAD

**Discussion by JakeTrock** [community/community#140148] [GitHub Education] [answered]
### Hi everyone,

I recently added my student email to my github account and was accepted into the github for students portal, when I go to the [student developer pack](https://education.github.com/pack), it says I have a copilot subscription, however when I go to [my profile](https://github.com/settings/copilot?editor=copilot_1.234.0) or vscode it says I do not, this is even after my school email has been set to my primary email. Please let me know if I missed a step in there.

<img width="900" alt="Screenshot 2024-09-29 at 7 14 34 PM" src="https://github.com/user-attachments/assets/a134818f-d91f-4d0a-9d7d-fc3ba436c98b">

<img width="935" alt="image" src="https://github.com/user-attachments/assets/26818b49-683d-4add-81e6-5d706201dc55">
<img width="38" alt="image" src="https://github.com/user-attachments/assets/0cb58d96-ebe1-4c16-b251-5e50d992bc33">

**Comment by davevad93** [comment, marked as answer]
Hi @JakeTrock ,

You should have access to your `Copilot` subscription indeed. First try to redeem your free copilot offer [here](https://education.github.com/pack/redeem/copilot-student), if you still can't redeem it, contact GitHub support by opening a ticket [here](https://support.github.com/contact/education), select "`I am a verified student having trouble redeeming my Global Campus offers`" and explain your issue, they will help you solve it.

**Reply by JakeTrock** [reply]
Look at that, just needed that one step. Thanks for the quick reply!

**Reply by davevad93** [reply]
You're welcome @JakeTrock , glad to help! 👍
